app/assets/stylesheets/views/_landing_page/quote.scss

```diff
@@ -0,0 +1,27 @@
+@import "govuk_publishing_components/individual_component_support";
+
+.app-b-quote {
+  color: govuk-colour("white");
```

Comment:
I guess at some point we might need the concept of `inverse` things here (as obviously this block would render badly if it wasn't on a dark background).